src/styles/stars.scss

```diff
@@ -0,0 +1,50 @@
+.stars {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+
+  &__container {
+    margin-bottom: 24px;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  .stars--4 &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
The BEM naming convention is not followed correctly here. Modifier classes should not be combined with element classes. Instead of nesting the modifier within the `.stars` block, create a separate modifier class for the `&__star` element, like `.stars__star--active`, and apply it to the relevant stars.